src/lib/metrics/regression.ts

```diff
@@ -29,12 +29,9 @@ export function mean_absolute_error(
   },
 ): number {
   const yTrueShape = inferShape(y_true);
```

Comment:
I think that this variable is not necessary anymore

Reply:
done